core/src/main/java/org/apache/gravitino/audit/v2/CompatibilityUtils.java

```diff
@@ -0,0 +1,110 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.apache.gravitino.audit.v2;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.gravitino.audit.AuditLog.Operation;
+import org.apache.gravitino.audit.AuditLog.Status;
+import org.apache.gravitino.listener.api.event.OperationStatus;
+import org.apache.gravitino.listener.api.event.OperationType;
+
+public class CompatibilityUtils {
+
+  private static Map<OperationType, Operation> operationTypeMap = new HashMap<>();
```

Comment:
You can directly use immutable map and initialize the immutable here, no need to use a static method and static block here.

Reply:
You mean use immutableMap.of(xxx) to initialize the map? There are number limits to the entries of the map.

Reply:
Use immutablemap's build method.

Reply:
updated